include/rang.hpp

```diff
@@ -88,7 +89,13 @@ enum class bgB {
 	gray    = 107
 };
 
-enum class control { autoColor = 0, forceColor = 1 };
+enum class control {
+    autoColor = 0,
+    forceColor = 1,
+    bufferInput = 2,
```

Comment:
are we using `bufferInput = 2, immediateInput = 3, rawInput = 4` yet or are these for something else?